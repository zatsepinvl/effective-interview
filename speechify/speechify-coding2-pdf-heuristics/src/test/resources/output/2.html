            <html><body>
            <div style="display: flex; flex-direction: row">
            <div style="width: 50%">
            <h1>Expected</h1>
            <hr />
            <p><i>THE MUSCLE & STRENGTH PYRAMID: NUTRITION</i></p>
<p>So what that means is that yes, there are going to be people who need 2600 calories to maintain 200 lb (not very much for a person that size), all the way up to maybe 4400 calories (a lot for a person that size). Most will fall in the middle range, but some won’t. So there really is a large variation, and there are people who are even further outside of those ranges believe it or not. Below, I show how to use the activity multiplier, notice there is a range at each level to represent individual differences.</p>
<h2>Step 2: Using An Activity Multiplier</h2>
<p>LIFESTYLE & TRAINING FREQUENCY</p>
<p>ACTIVITY MULTIPLIER</p>
<p>Sedentary plus 3-6 days of weight lifting</p>
<p>1.3 – 1.6</p>
<p>Lightly active plus 3-6 days of weight lifting</p>
<p>1.5 – 1.8</p>
<p>Active plus 3-6 days of weight lifting</p>
<p>1.7 – 2.0</p>
<p>Very active plus 3-6 days of weight lifting</p>
<p>1.9 – 2.2</p>
<p>Baseline Multiplier x Activity Multiplier = Estimated Calories For Body weight Maintenance</p>
<p>Depending on lifestyle and individual differences, these calculations can equate to a 2600 – 4400 calorie range for maintenance in our 200 lb (90 kg) male example.</p>
<p>Now I know what you are wondering: “How do I know which value to choose, what determines each level of activity?” Well, that’s why it is best to actually take the two weeks to fully track your intake as discussed earlier. However, if you want to get started faster, just take the middle value of 1.7 as your multiplier. Then, if you gain or lose weight too quickly after 2–3 weeks, just adjust your intake. This is a perfectly valid approach as well.</p>
<p><i>ENERGY BALANCE</i></p>
<p><i>48</i></p>
            </div>
            <div style="width: 50%">
            <h1>Actual</h1>
            <hr />
            
            </div>
            </div>
            </body></html>