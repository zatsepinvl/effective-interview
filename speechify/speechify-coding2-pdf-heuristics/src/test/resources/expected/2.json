[
  { "type": "Marginalia", "text": "THE MUSCLE & STRENGTH PYRAMID: NUTRITION" },
  {
    "type": "Paragraph",
    "text": "So what that means is that yes, there are going to be people who need 2600 calories to maintain 200 lb (not very much for a person that size), all the way up to maybe 4400 calories (a lot for a person that size). Most will fall in the middle range, but some won\u2019t. So there really is a large variation, and there are people who are even further outside of those ranges believe it or not. Below, I show how to use the activity multiplier, notice there is a range at each level to represent individual differences."
  },
  { "type": "Heading", "text": "Step 2: Using An Activity Multiplier" },
  { "type": "Paragraph", "text": "LIFESTYLE & TRAINING FREQUENCY" },
  { "type": "Paragraph", "text": "ACTIVITY MULTIPLIER" },
  { "type": "Paragraph", "text": "Sedentary plus 3-6 days of weight lifting" },
  { "type": "Paragraph", "text": "1.3 \u2013 1.6" },
  {
    "type": "Paragraph",
    "text": "Lightly active plus 3-6 days of weight lifting"
  },
  { "type": "Paragraph", "text": "1.5 \u2013 1.8" },
  { "type": "Paragraph", "text": "Active plus 3-6 days of weight lifting" },
  { "type": "Paragraph", "text": "1.7 \u2013 2.0" },
  {
    "type": "Paragraph",
    "text": "Very active plus 3-6 days of weight lifting"
  },
  { "type": "Paragraph", "text": "1.9 \u2013 2.2" },
  {
    "type": "Paragraph",
    "text": "Baseline Multiplier x Activity Multiplier = Estimated Calories For Body weight Maintenance"
  },
  {
    "type": "Paragraph",
    "text": "Depending on lifestyle and individual differences, these calculations can equate to a 2600 \u2013 4400 calorie range for maintenance in our 200 lb (90 kg) male example."
  },
  {
    "type": "Paragraph",
    "text": "Now I know what you are wondering: \u201cHow do I know which value to choose, what determines each level of activity?\u201d Well, that\u2019s why it is best to actually take the two weeks to fully track your intake as discussed earlier. However, if you want to get started faster, just take the middle value of 1.7 as your multiplier. Then, if you gain or lose weight too quickly after 2\u20133 weeks, just adjust your intake. This is a perfectly valid approach as well."
  },
  { "type": "Marginalia", "text": "ENERGY BALANCE" },
  { "type": "Marginalia", "text": "48" }
]

